etc/profile-a-l/goldendict.profile

```diff
@@ -0,0 +1,62 @@
+# Firejail profile for goldendict
+# This file is overwritten after every install/update
+# Persistent local customizations
+include default.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.goldendict
+noblacklist ${HOME}/.cache/GoldenDict
+# Allow donwloading image from entries
+noblacklist ${DOWNLOADS}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.goldendict
+mkdir ${HOME}/.cache/GoldenDict
+whitelist ${HOME}/.goldendict
+whitelist ${HOME}/.cache/GoldenDict
+whitelist ${DOWNLOADS}
+# The default path of dictionaries
+whitelist /usr/share/stardict/dic
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
+netfilter
+# no3d leads to the libGL MESA-LOADER errors
+#no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noroot
+notv
```

Comment:
`nosound`?

Reply:
sound is required for pronunciation audios

Reply:
Then remove `machine-id` and add `machine-id` to `private-etc`.

Reply:
For this application, machine-id does not broke the audio output. That is why I kept it.

Reply:
It does not depend on the application, it depends on the sound-server.

Reply:
`machine-id` breaks pulseaudio.
PipeWire, pipewire-pulse, Jack and "raw-ALSA" work with `machine-id`.

Reply:
While, my environment is pulseaudio. I have no idea why it works. But anyway, I agree to move it to `private-etc` just in case of any uncertainties.